### PR TITLE
Update delta-lake-interoperability.md

### PR DESCRIPTION
**Pull request by anshulsharmas**
Added a note for retention capability in KQL databases.

# Thank you for contributing to Microsoft Fabric documentation

## Fill out these items before submitting your pull request:

If you are working internally at Microsoft:
**Provide a link to an Azure DevOps Boards work item that tracks this feature/update.**

>
>

**Who is your primary Skilling team contact?** \@mention them individually tag them and let them review the PR before signing off.

>
>

## For internal Microsoft contributors, check off these quality control items as you go

- [ ] 1. **Check the Acrolinx report:** Make sure your Acrolinx Total score is **above 80 minimum** (higher is better) and with no spelling issues. Acrolinx ensures we are providing consistent terminology and using an appropriate voice and tone, and helps with localization.

- [ ] 2. **Successful build with no warnings or suggestions**: Review the build status to make sure **all files are green** (Succeeded).

- [ ] 3. **Preview the pages:**: Click each **Preview URL** link to view the rendered HTML pages on the review.learn.microsoft.com site to check the formatting and alignment of the page. Scan the page for overall formatting, and look at the parts you edited in detail.

- [ ] 4. **Check the Table of Contents:** If you are adding a new markdown file, make sure it is linked from the table of contents.

- [ ] 5. **#sign-off to request PR review and merge**: Once the pull request is finalized and ready to be merged, indicate so by typing `#sign-off` in a new comment in the Pull Request. If you need to cancel that sign-off, type `#hold-off` instead. *Signing off means the document can be published at any time.* Note, this is a formatting and standards review, not a technical review.

## Merge and publish
- After you `#sign-off`, there is a separate PR Review team that will review the PR and describe any necessary feedback before merging. 
- The review team will use the comments section in the PR to provide feedback if changes are needed. Address any blocking issues and sign off again to request another review.
- Once all feedback is resolved, you can `#sign-off` again. The PR Review team reviews and merges the pull request into the specified branch (usually the main branch or a release- branch).
- From the main branch, the change is merged into the live branch several times a day to publish it to the public learn.microsoft.com site.

### CODE REVIEW

File: docs/get-started/delta-lake-interoperability.md [hunk 1]
```diff
@@ -38,13 +38,15 @@ Dataflows|Yes|Yes|Yes|No|Yes|Yes|Reader: 1<br/>Writer: 2<br/>|
...
 > * Some Fabric experiences do not have inherited table optimization and maintenance capabilities, such as bin-compaction, V-order, and clean up of old unreferenced files. To keep Delta Lake tables optimal for analytics, follow the techniques in [Use table maintenance feature to manage delta tables in Fabric](../data-engineering/lakehouse-table-maintenance.md) for tables ingested using those experiences.
+> * Some Fabric experiences do not have inherited table optimization and maintenance capabilities, such as bin-compaction, V-order, and clean up of old unreferenced files. To keep Delta Lake tables optimal for analytics, follow the techniques in [Use table maintenance feature to manage delta tables in Fabric](../data-engineering/lakehouse-table-maintenance.md) for tables ingested using those experiences.
+> * #KQL databases provide certain table maintenance capabilities such as [retention](../real-time-analytics/data-policies), data is removed at the end of retention period from OneLake. Refer [One Logical copy](../real-time-analytics/one-logical-copy).
```
Comment (deemadiab) [inline docs/get-started/delta-lake-interoperability.md:49]:
```suggestion
> *  KQL databases provide certain table maintenance capabilities such as [retention](../real-time-analytics/data-policies.md). Data is removed at the end of the retention period from OneLake. For more information, see [One Logical copy](../real-time-analytics/one-logical-copy.md).
```

**Comment by deemadiab** [comment]
#sign-off

**Comment by anshulsharmas** [comment]
#sign-off

File: docs/get-started/delta-lake-interoperability.md [hunk 2]
```diff
@@ -38,13 +38,15 @@ Dataflows|Yes|Yes|Yes|No|Yes|Yes|Reader: 1<br/>Writer: 2<br/>|
 Data pipelines|No|No|Yes|No|Yes, overwrite only|Yes|Reader: 1<br/>Writer: 2|
 Power BI direct lake semantic models|Yes|Yes|N/A (not applicable)|N/A (not applicable)|N/A (not applicable)|Yes|N/A (not applicable)|
 Export Power BI semantic models into OneLake|Yes|N/A (not applicable)|Yes|No|Yes|N/A (not applicable)|Reader: 2<br/>Writer: 5|
-KQL databases|Yes|Yes|No|No|Yes|Yes|Reader: 1<br/>Writer: 1|
+KQL databases|Yes|Yes|No|No#|Yes|Yes|Reader: 1<br/>Writer: 1|
```
Comment (deemadiab) [inline docs/get-started/delta-lake-interoperability.md:41]:
```suggestion
KQL databases|Yes|Yes|No|No|Yes|Yes|Reader: 1<br/>Writer: 1|
```

File: docs/get-started/delta-lake-interoperability.md [hunk 3]
```diff
@@ -38,13 +38,15 @@ Dataflows|Yes|Yes|Yes|No|Yes|Yes|Reader: 1<br/>Writer: 2<br/>|
...
 > * Some Fabric experiences do not have inherited table optimization and maintenance capabilities, such as bin-compaction, V-order, and clean up of old unreferenced files. To keep Delta Lake tables optimal for analytics, follow the techniques in [Use table maintenance feature to manage delta tables in Fabric](../data-engineering/lakehouse-table-maintenance.md) for tables ingested using those experiences.
+> * Some Fabric experiences do not have inherited table optimization and maintenance capabilities, such as bin-compaction, V-order, and clean up of old unreferenced files. To keep Delta Lake tables optimal for analytics, follow the techniques in [Use table maintenance feature to manage delta tables in Fabric](../data-engineering/lakehouse-table-maintenance.md) for tables ingested using those experiences.
+> * #KQL databases provide certain table maintenance capabilities such as [retention](../real-time-analytics/data-policies.md). Data is removed at the end of the retention period from OneLake. For more information, see [One Logical copy](../real-time-analytics/one-logical-copy.md).
```
Comment (deemadiab) [inline docs/get-started/delta-lake-interoperability.md:49]:
```suggestion
> * KQL databases provide certain table maintenance capabilities such as [retention](../real-time-analytics/data-policies.md). Data is removed at the end of the retention period from OneLake. For more information, see [One Logical copy](../real-time-analytics/one-logical-copy.md).
```

**Review by deemadiab — COMMENTED** [review]
Suggestion to remove footnote reference.

File: docs/get-started/delta-lake-interoperability.md (hunk 3 above)
Comment (deemadiab) [inline docs/get-started/delta-lake-interoperability.md:49]:
```suggestion

<sup>*</sup> KQL databases provide certain table maintenance capabilities such as [retention](../real-time-analytics/data-policies.md). Data is removed at the end of the retention period from OneLake. For more information, see [One Logical copy](../real-time-analytics/one-logical-copy.md).

> [!NOTE]
>
> * Fabric doesn't write name-based column mappings by default. The default Fabric experience generates tables that are compatible across the service. Delta lake, produced by third-party services, may have incompatible table features.
> * Some Fabric experiences do not have inherited table optimization and maintenance capabilities, such as bin-compaction, V-order, and clean up of old unreferenced files. To keep Delta Lake tables optimal for analytics, follow the techniques in [Use table maintenance feature to manage delta tables in Fabric](../data-engineering/lakehouse-table-maintenance.md) for tables ingested using those experiences.
> * Some Fabric experiences do not have inherited table optimization and maintenance capabilities, such as bin-compaction, V-order, and clean up of old unreferenced files. To keep Delta Lake tables optimal for analytics, follow the techniques in [Use table maintenance feature to manage delta tables in Fabric](../data-engineering/lakehouse-table-maintenance.md) for tables ingested using those experiences.
```

**Review by deemadiab — COMMENTED** [review]
Moving placement of footnote to comply with style guide.

**Pull request closed without merging**